src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/AuthorizationServer/ApiAccess/form.html.twig

```diff
@@ -23,31 +23,66 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
 
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+
 {% form_theme apiAccessForm '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
 
-{% block api_access_form_block %}
-  {{ form_start(apiAccessForm) }}
+{% block session_alert %}
+  {{ parent() }}
+
+  {# Display generated client secrets passed ny session #}
+  {% if app.session.flashbag.peek('client_secret')|length() > 0 %}
+    {% for clientSecret in app.session.flashbag.get('client_secret') %}
+      <div class="alert alert-success d-print-none" role="alert">
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+          <span aria-hidden="true"><i class="material-icons">close</i></span>
+        </button>
+        <div class="alert-text">
+          {{ 'Client secret:'|trans({}, 'Admin.Advparameters.Feature') }} {{ clientSecret }}
+          <a href="#" class="copy-secret-to-clipboard" data-secret="{{ clientSecret|trim }}">
+            {{ 'Copy'|trans({}, 'Admin.Actions') }}
+            <i class="material-icons">content_copy</i>
+          </a>
+        </div>
+      </div>
+    {% endfor %}
+  {% endif %}
+{% endblock %}
+
+{% block content %}
+  {% block api_access_form_block %}
+    {{ form_start(apiAccessForm) }}
 
-  <div class="card">
-    <h3 class="card-header">
-      {{ 'API Access'|trans({}, 'Admin.Advparameters.Feature') }}
-    </h3>
+    <div class="card">
+      <h3 class="card-header">
+        {{ 'API Access'|trans({}, 'Admin.Advparameters.Feature') }}
+      </h3>
 
-    <div class="card-body">
-      <div class="form-wrapper">
-        {{ form_widget(apiAccessForm) }}
+      <div class="card-body">
+        <div class="form-wrapper">
+          {{ form_widget(apiAccessForm) }}
+        </div>
       </div>
-    </div>
 
-    <div class="card-footer">
-      <a href="{{ path('admin_api_accesses_index') }}" class="btn btn-outline-secondary">
-        {{ 'Cancel'|trans({}, 'Admin.Actions') }}
-      </a>
-      <button class="btn btn-primary float-right">
-        {{ 'Save'|trans({}, 'Admin.Actions') }}
-      </button>
+      <div class="card-footer">
+        <a href="{{ path('admin_api_accesses_index') }}" class="btn btn-outline-secondary">
+          {{ 'Cancel'|trans({}, 'Admin.Actions') }}
+        </a>
+        <div class="float-right">
+          {% block form_submit_buttons %}
+            <button type="submit" class="btn btn-primary">
+              {{ 'Generate client secret and save'|trans({}, 'Admin.Advparameters.Feature') }}
```

Comment:
It looks like a CTA, shouldn't it be in _Admin.Actions_?

Reply:
It's the label of the submit button in the creation form. I though `Admin.Actions` only contained generic wording that are used in many different places with different scopes. That's why I preferred the namespace related to advanced parameters because the wording is too specific to be used anywhere else.

But maybe I didn't understand the purpose of the `Admin.Actions` so let me know which one is best

![Capture d’écran 2023-10-27 à 10 35 41](https://github.com/PrestaShop/PrestaShop/assets/13801017/0ccba8d1-1e20-4eed-a3bc-afce628b4f61)

Reply:
You're totally right, sorry it's been a long time since I did a wording review. 😅